src/Entity/Geo/Country.php

```diff
@@ -39,12 +39,14 @@ public function setForeignDistrict(?ForeignDistrict $foreignDistrict): void
 
     public function getParents(): array
     {
-        $toMerge = [
-            [$this->foreignDistrict],
-            $this->foreignDistrict->getParents(),
-        ];
+        $toMerge = [];
 
-        return array_values(array_unique(array_merge(...$toMerge)));
+        if ($this->foreignDistrict) {
+            $toMerge[] = [$this->foreignDistrict];
+            $toMerge[] = $this->foreignDistrict->getParents();
+        }
+
+        return $toMerge ? array_values(array_unique(array_merge(...$toMerge))) : [];
```

Comment:
as an alternative:
```php
if (!$this->foreignDistrict) {
    return [];
}

return array_values(array_unique(array_merge(
    [$this->foreignDistrict],
    $this->foreignDistrict->getParents()
)));
```

Reply:
Thanks for the feedback.
I'll keep the filling strategy to be coherent with other implementations.